src/styles/main.scss

```diff
@@ -1,3 +1,67 @@
+@import 'variables';
+
+@font-face {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
   margin: 0;
 }
+
+.calendar-block {
+  display: flex;
+  justify-content: left;
+  align-items: center;
+  gap: $day-gap;
+
+  width: $day-size * 7 + $day-gap * 6 + $block-padding * 2;
+  padding: $block-padding;
+
+  flex-wrap:wrap;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    border: 1px solid $color-border;
+    width: $day-size;
+    height: $day-size;
+    box-sizing: border-box;
```

Comment:
move to the `*` selector
```suggestion
```

Reply:
Hello, i understand your position, but: 
- 1. You should get to use make better from the start
- 2. If you are working on a large project, then, most likely, there are other developers working with you. Accordingly, every developer must understand that the code works predictably. But here one block is allocated one by one, and in another case, one block at a time - it's inconvenient